frontend/src/store/eam-worker-configurate/actions.ts

```diff
@@ -31,4 +32,13 @@ const workerCreate = createAsyncThunk<
   },
 );
 
-export { workerCreate };
+const getWorkers = createAsyncThunk<
+  EAMWorkerGetAllResponseDto,
+  void,
+  AsyncThunkConfig
+>(ActionType.GET_WORKERS, async (_payload, { extra }) => {
+  const { workerApi } = extra;
+  return await workerApi.getAll();
```

Comment:
```suggestion
  return workerApi.getAll();
```
`frontend/src/store/eam-worker-configurate/actions.ts` -- this is for eam-worker-configurate
you need the same action for your page in your scope